src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+.card {
+  font-family: $main-font-family;
+  @include text(#616070, 10px, 400);
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    padding: 0 3px;
```

Comment:
don't set paddings for image, just move it to the center of the card

Reply:
Fixed.